ledger-api/grpc-definitions/com/daml/ledger/api/v1/command_submission_service.proto

```diff
@@ -31,6 +32,9 @@ service CommandSubmissionService {
   // Submit a single composite command.
   rpc Submit (SubmitRequest) returns (google.protobuf.Empty);
 
+  // Submit a single transfer.
+  rpc SubmitTransfer (SubmitTransferRequest) returns (google.protobuf.Empty);
```

Comment:
I thin we should never again introduce grpc calls that return `google.protobuf.Empty`. We have a couple of examples clustered in this service, but I think it is a mistake. It is painting us into a corner, where this method will have to return an empty forever.

Reply:
Good idea :+1: